org.sebicodes.Prt.yml

```diff
@@ -0,0 +1,31 @@
+app-id: org.sebicodes.Prt
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: runner.sh
+modules:
+  - name: app
+    buildsystem: simple
+    build-commands:
+      - install -D runner.sh /app/bin/runner.sh
+      - install -D app.py /app/bin/app.py
+      - install -D .desktop /app/share/applications/com.sebicodes.Prt.desktop
+      - install -D plruth.png /app/share/icons/hicolor/500x500/apps/com.sebicodes.Prt.png
+    sources:
+       - type: archive
+         url: https://raw.githubusercontent.com/Sebicodes99/pleaserunthat-beta/stable/Flatpak/.desktop
+         sha256: f65851539d48b97ef3d295a342ac2b09449d0f3e1d483e641e49499511f94325
+       - type: archive
+         path: https://raw.githubusercontent.com/Sebicodes99/pleaserunthat-beta/stable/app.py
+         sha256: 822fbcd41b88c4b1d17d324a4c0690afe190a16dd938b2488c5a2816cb6fff91
+       - type: archive
+         path: https://raw.githubusercontent.com/Sebicodes99/pleaserunthat-beta/stable/Flatpak/runner.sh
+         sha256: a333c84a33aa7671be1adf8ea8d8ee067ae922adfc7f08683d19830cffbda838
+       - type: archive
+         path: https://github.com/Sebicodes99/pleaserunthat-beta/blob/stable/Flatpak/plruth.png?raw=true
+         sha256: 06b636ef882633bf779158488df8b396f91b06c3d2401078648d201ef750559a
+       - type: archive
+         path: https://raw.githubusercontent.com/Sebicodes99/pleaserunthat-beta/stable/Flatpak/org.sebicodes.Prt.appdata.xml
+         sha256: 3e8d8375df499c97561678b73f0c92556268ec321c537d9c53438e81e531c1c9
+finish-args:
```

Comment:
finish args is usually above `modules`.

also the app doesn't display any GUI?

Reply:
No, It's a terminal app. .desktop includes it.

Reply:
I will change it

Reply:
Done

Reply:
CLI apps really aren't the best use case for flatpak and flathub. Just setting the expectations here.